_maps/shuttles/syndicate/syndicate_litieguai.dmm

```diff
@@ -1,989 +1,1108 @@
+	dir = 1
 	},
 /obj/machinery/door/firedoor/border_only,
-/obj/effect/turf_decal/industrial/warning,
-/obj/effect/turf_decal/industrial/warning{
-	dir = 1
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/machinery/atmospherics/pipe/simple/supply/hidden/layer2,
+/obj/machinery/door/airlock/medical{
+	name = "Surgery"
 	},
-/obj/machinery/door/poddoor/shutters{
-	id = "traumalobby";
-	name = "Lobby"
+/obj/structure/cable{
+	icon_state = "1-2"
 	},
-/turf/open/floor/engine,
-/area/ship/cargo)
-"hq" = (
-/turf/open/floor/plasteel/stairs/right{
+/turf/open/floor/plasteel/tech/grid,
+/area/ship/medical/surgery)
+"fp" = (
+/turf/closed/wall/mineral/titanium,
+/area/ship/maintenance/port)
+"fW" = (
+/obj/effect/turf_decal/spline/fancy/opaque/black{
+	dir = 4
+	},
+/obj/machinery/holopad/emergency/medical,
+/obj/machinery/atmospherics/pipe/simple/scrubbers/hidden/layer4,
+/obj/effect/turf_decal/spline/fancy/opaque/black{
 	dir = 8
 	},
-/area/ship/storage)
-"hv" = (
-/obj/effect/turf_decal/arrows/red{
+/obj/structure/cable{
+	icon_state = "1-2"
+	},
+/turf/open/floor/plasteel/patterned/brushed,
+/area/ship/medical)
+"fX" = (
+/obj/machinery/vending/snack/random,
+/obj/effect/turf_decal/trimline/opaque/red/filled/line{
 	dir = 8
 	},
-/obj/effect/turf_decal/spline/fancy/opaque/black{
-	dir = 4
+/obj/structure/sign/poster/official/cleanliness{
+	pixel_y = 32
 	},
-/turf/open/floor/plasteel/patterned/grid,
+/turf/open/floor/plasteel/white,
+/area/ship/cargo)
+"gf" = (
+/obj/effect/turf_decal/industrial/outline/red,
+/obj/structure/closet/crate/freezer/surplus_limbs,
+/obj/machinery/airalarm/directional/south,
+/obj/machinery/firealarm/directional/east,
+/turf/open/floor/plasteel/tech,
 /area/ship/storage)
+"gL" = (
+/turf/closed/wall/mineral/titanium/nodiagonal,
+/area/ship/cargo)
+"gW" = (
+/obj/effect/turf_decal/trimline/opaque/red/filled/line,
+/turf/open/floor/plasteel/white,
+/area/ship/hallway/central)
 "hF" = (
 /obj/structure/table/reinforced,
 /obj/machinery/fax,
 /turf/open/floor/plasteel/dark,
 /area/ship/bridge)
-"hH" = (
-/obj/machinery/computer/helm,
-/turf/open/floor/plasteel/dark,
-/area/ship/bridge)
 "hQ" = (
 /obj/structure/table/optable,
-/obj/structure/window/reinforced{
-	dir = 1
+/turf/open/floor/plasteel/tech,
+/area/ship/medical/surgery)
+"hS" = (
+/obj/effect/turf_decal/industrial/hatch/red,
+/obj/machinery/firealarm/directional/west,
+/obj/machinery/light_switch{
+	dir = 4;
+	pixel_x = -20;
+	pixel_y = -10
 	},
-/turf/open/floor/plasteel/tech/grid,
-/area/ship/medical)
-"hT" = (
-/obj/machinery/power/apc/auto_name/directional/south,
-/obj/effect/turf_decal/siding/white/end,
-/obj/structure/cable,
-/obj/machinery/atmospherics/components/unary/vent_pump/on/layer2{
-	dir = 1
+/obj/item/reagent_containers/glass/bottle/formaldehyde,
+/obj/item/storage/box/syringes,
+/obj/item/storage/box/bodybags,
+/obj/structure/closet/secure_closet{
+	icon_state = "med";
+	name = "medicine locker"
```

Comment:
This should be a plain grey locker named "mortuary locker" for consistency with other medships.